ui/src/adapters/api/credentials.ts

```diff
@@ -133,15 +199,15 @@ export async function getCredentials({
       },
       method: "GET",
       params: new URLSearchParams({
-        ...(did !== undefined ? { did } : {}),
+        ...(credentialSubject !== undefined ? { credentialSubject } : {}),
```

Comment:
This is wrong. @martinsaporiti @x1m3 can you please clarify why this param has been renamed?

Reply:
[Slack discussion on the topic](https://privadoid.slack.com/archives/C074GNHEN82/p1728633454050889)

Reply:
As decided in the the thread, we leave it as it's.